CHANGELOG.md

```diff
@@ -6,14 +6,12 @@ numbers of H3-Py are the major and minor version of the bound core library,
 respectively. The patch version is incremented independently of the core
 library.
 
+We use [this changelog structure](http://keepachangelog.com/).
+
 Because H3-Py is versioned in lockstep with the H3 core library, please
 avoid adding features or APIs which do not map onto the
 [H3 core API](https://uber.github.io/h3/#/documentation/api-reference/).
 
-# Change Log
-
-All notable changes to this project will be documented in this file.
-This project adheres to [Semantic Versioning](http://semver.org/) and uses [this changelog structure](http://keepachangelog.com/).
 
```

Comment:
Add a changelog entry in this PR? Or when publishing after this PR?

Reply:
updated!